app/models/course/achievement.rb

```diff
@@ -10,6 +10,7 @@ class Course::Achievement < ActiveRecord::Base
                                       inverse_of: :achievement, dependent: :destroy
   has_many :course_users, through: :course_user_achievements, class_name: CourseUser.name
 
+  accepts_nested_attributes_for :course_users, allow_destroy: true
```

Comment:
Will remove this. Thought this was needed to update `@achievement.course_user_ids` >_<